gettingStarted/sysElevenStackKickstart.yaml

```diff
@@ -16,30 +16,44 @@ description: Simple template to deploy a single compute instance
 
 parameters:
  key_name:
-  type: string
+   type: string
+   default: 
+ username:
+   type: string
+   default: syseleven
+ instance_name:
+   type: string
+   default: kickstart
+ image:
+   type: string
+   default: Ubuntu Server 16.04 LTS
+ flavor:
+   type: string
+   default: m1.micro
  public_network:
-  type: string
-  default: ext-net 
+   type: string
+   default: ext-net
 
 resources:
   allow_ssh:
     type: OS::Neutron::SecurityGroup
     properties:
       description: allow incoming SSH and ICMP traffic from anywhere.
-      name: allow incoming traffic, tcp port 22 and icmp
+      name: 
+        list_join: [ '-', [ { get_param: instance_name }, 'allow incoming traffic, tcp port 22 and icmp']]
       rules: 
         - { direction: ingress, remote_ip_prefix: 0.0.0.0/0, port_range_min: 22, port_range_max: 22, protocol: tcp } 
         - { direction: ingress, remote_ip_prefix: 0.0.0.0/0, protocol: icmp } 
 
   host:
     type: OS::Nova::Server
     properties:
-      name: kickstart
+      name: {get_param: instance_name }
```

Comment:
Let's say the customer called the stack "test", it would be nice if the server is called "test-kickstart" in the end

Reply:
see commit 55e78290a0cba68dfb6b7c3248faa0cdf294eb92